monai/data/decathlon_datalist.py

```diff
@@ -148,3 +149,35 @@ def load_decathlon_properties(data_property_file_path: str, property_keys: Union
             raise KeyError(f"key {key} is not in the data property file.")
         properties[key] = json_data[key]
     return properties
+
+
+def check_missing_files(datalist: List[Dict], keys: KeysCollection, allow_missing_keys: bool = False):
+    """Checks whether some files in the Decathlon datalist are missing.
+    It would be helpful to check missing files before a heavy training.
+
+    Args:
+        datalist: a list of data items, every item is a dictionary.
+            ususally generated by `load_decathlon_datalist` API.
+        keys: expected keys to check in the datalist.
+        allow_missing_keys: whether allow missing keys in the datalist items.
+            if False, raise exception if missing. default to False.
+
+    Returns:
+        A list of missing file name.
+
+    """
+    missing_files = []
+    for item in datalist:
+        for k in ensure_tuple(keys):
+            if k not in item:
+                if not allow_missing_keys:
+                    raise ValueError(f"key `{k}` is missing in the datalist item: {item}")
+                continue
+
+            for f in ensure_tuple(item[k]):
+                if not isinstance(f, str):
+                    raise ValueError(f"filepath of key `{k}` must be a string or a list of strings, but got: {f}.")
+                if not os.path.exists(f):
```

Comment:
Is it a concern if `f` isn't an absolute path but a relative path? This would be checking that `f` exists relative to the current directory, maybe want to use `abspath` before checking it exists. The paths put into `missing_files` perhaps should be absolute as well.

Reply:
Sure, thanks for your great suggestion!
That's a good point, I added support in the latest commit.

Thanks.